mappings/net/minecraft/world/gen/PositionalRandomFactory.mapping

```diff
@@ -1 +1,13 @@
 CLASS net/minecraft/unmapped/C_coqzjjrt net/minecraft/world/gen/PositionalRandomFactory
+	METHOD m_bpedoycx createRandomAt (Lnet/minecraft/unmapped/C_hynzadkk;)Lnet/minecraft/unmapped/C_rlomrsco;
```

Comment:
what does this do that the other createRandom methods don't if I can ask?

Reply:
It creates a Random instance using the provided block pos as seed